src/libs/protocols/protos/confidential_data_hub.proto

```diff
@@ -28,6 +28,24 @@ message SecureMountResponse {
     string mount_path = 1;
 }
 
+message ImagePullRequest {
+    // - `image_url`: The reference of the image to pull
+    string image_url = 1;
+
+    // - `bundle_path`: The path to store the bundle. After the RPC is
+    // called, Confidential Data Hub will put a `config.json` file
+    // and a mounted `rootfs` directory under the this path. Note
+    // that usually the caller and the server (CDH) runs on the same
+    // machine, so the the filesystem of the client and the server are
+    // the same. The bundle path is relative to the server's filesystem.
```

Comment:
The term `relative` may be misleading here (as in: relative paths). Maybe reword as "The bundle path is an absolute path referring to the client's root filesystem"?